pkgs/development/tools/misc/arcanist/default.nix

```diff
@@ -29,26 +30,37 @@ stdenv.mkDerivation {
     rev = "2565cc7b4d1dbce6bc7a5b3c4e72ae94be4712fe";
     sha256 = "0jiv4aj4m5750dqw9r8hizjkwiyxk4cg4grkr63sllsa2dpiibxw";
   };
-  buildInputs = [ bison flex php ];
+
+  buildInputs = [ php ];
+
+  nativeBuildInputs = [ bison flex installShellFiles ];
 
   postPatch = lib.optionalString stdenv.isAarch64 ''
     substituteInPlace support/xhpast/Makefile \
       --replace "-minline-all-stringops" ""
   '';
 
   buildPhase = ''
+    runHook preBuild
     make cleanall -C support/xhpast
     make xhpast -C support/xhpast
```

Comment:
Missing $makeFlags/$NIX_BUILD_CORES.

Reply:
should I add those in this pull request, or would a separate cleanup change be better?

Reply:
Just do it in this PR. We are cleaning up anyway and this saves a hydra rebuild cycle.

Reply:
cool, coming right up.